<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stu_Select</class>
 <widget class="QWidget" name="Stu_Select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:QLinearGradient(x1:0,y1:0,x2:0,y2:1,stop:0 #efefff,stop:1 #cfcfff);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>251</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>课程号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sectionnumEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>216</width>
        <height>43</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>输入想选课程号并按确定键</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="selectButton_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>500</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>点击查询查看可选课程信息</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>351</width>
        <height>431</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>课程号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>必修/选修</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>教师</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学分</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>已选/可选</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>剩余人数</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>500</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QPushButton" name="returnButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>500</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>返回学生主界面</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;》》》》》》选课界面《《《《《《&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
